Makefile

```
@@ -51,6 +51,10 @@ client: common
 server: common
 	$(VENV_SCRIPT)python server
 
+.PHONY: test-common
+tests: common
```

```suggestion
.PHONY: test-common
tests: common
```
It is `test-common` or `tests`? You may choose only one